src/containerapp/azext_containerapp/custom.py

```diff
@@ -2646,6 +2651,57 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+def create_or_update_java_logger(cmd, logger_name, logger_level, name, resource_group_name, no_wait=False):
+    raw_parameters = locals()
+
+    containerapp_java_logger_decorator = ContainerappJavaLoggerDecorator(
+        cmd=cmd,
+        client=ContainerAppPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+
+    containerapp_java_logger_decorator.validate_enabled_java_agent()
```

Comment:
`validate_argument()` will be more common to consistent with currently Decorator's design.